workers/unity/Assets/Gdk/Core/Tests/Editmode/Worker/WorkerRegistryTest.cs

```diff
@@ -27,10 +17,12 @@ public void OneTimeTearDown()
         [Test]
         public void SetWorkerForWorld_sets_worker_for_world_twice()
         {
-            var testWorker = new UnityTestWorker("someId", Vector3.zero);
-            testWorker.RegisterSystems();
-            var exception = Assert.Throws<System.Exception>(() => WorkerRegistry.SetWorkerForWorld(testWorker));
-            Assert.IsTrue(exception.Message.Contains("worker") && exception.Message.Contains("world"));
+            using (var testWorker = new UnityTestWorker("someId", Vector3.zero))
+            {
+                testWorker.RegisterSystems();
+                var exception = Assert.Throws<System.Exception>(() => WorkerRegistry.SetWorkerForWorld(testWorker));
+                Assert.IsTrue(exception.Message.Contains("worker") && exception.Message.Contains("world"));
```

Comment:
This test seems fragile - Just test against the literal expected message?

Reply:
I had only modified it to use the "using" block. I agree with your comment; but the team's opinion is that if we have the specific error message it will be a pain if we end up changing the exception message, we are testing that "There is some error about workers and worlds" and not the exception matches it specifically.

Reply:
Created UTY-551